commons/content-package-builder/src/main/java/io/wcm/tooling/commons/contentpackagebuilder/ContentPackage.java

```diff
@@ -85,11 +85,19 @@
     this.zip = new ZipOutputStream(os);
 
     this.transformerFactory = TransformerFactory.newInstance();
-    this.transformerFactory.setAttribute("indent-number", 2);
+    try {
+      this.transformerFactory.setAttribute("indent-number", 2);
+    } catch (IllegalArgumentException e) {
+        // Implementation does not support configuration property. Ignore.
+    }
     try {
       this.transformer = transformerFactory.newTransformer();
-      this.transformer.setOutputProperty(OutputKeys.INDENT, "yes");
-      this.transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
+      try {
+        this.transformer.setOutputProperty(OutputKeys.INDENT, "yes");
```

Comment:
I’m not sure if these really need to be wrapped in `try`…

* `"indent"` being in the `OutputKeys.INDENT` constant seems to suggest that it’s a default property that all implementations have to support.
* `{http://xml.apache.org/xslt}indent-amount` is namespaced and the [documentation for `setOutputProperty`](file:///home/rafi/.local/share/Zeal/Zeal/docsets/Java_SE8.docset/Contents/Resources/Documents/javax/xml/transform/Transformer.html#setOutputProperty-java.lang.String-java.lang.String-) tells us the exception is only thrown if “If the property is not supported, *and is not qualified with a namespace*.” (emphasis mine).

Still, better be safe than sorry, I’d say…